README.md

```diff
@@ -86,14 +86,16 @@ _This tutorial assumes that you have some minimal background of how to use the c
   * [13.2: Writing Complex Multisig Scripts](13_2_Writing_Complex_Multisig_Scripts.md)
   * [13.3: Empowering Bitcoin with Scripts](13_3_Empowering_Bitcoin_with_Scripts.md)
 
-### PART FOUR: USING TOR
+### PART FOUR: PRIVACY
 
 **Status:** Finished.
 
-* [14.0: Using Tor](14_0_Using_Tor.md)
-  * [14.1: Verifying Your Tor Setup](14_1_Verifying_Your_Tor_Setup.md)
-  * [14.2: Changing Your Bitcoin Hidden Services](14_2_Changing_Your_Bitcoin_Hidden_Services.md)
-  * [14.3: Adding SSH Hidden Services](14_3_Adding_SSH_Hidden_Services.md)
+* [14.1: Using Tor](14_1_Using_Tor.md)
+  * [14.1.1: Verifying Your Tor Setup](14_1_1_Verifying_Your_Tor_Setup.md)
+  * [14.1.2: Changing Your Bitcoin Hidden Services](14_1_2_Changing_Your_Bitcoin_Hidden_Services.md)
+  * [14.1.3: Adding SSH Hidden Services](14_1_3_Adding_SSH_Hidden_Services.md)
+
+* [14.2: Using i2p](14_2_Using_i2p.md)
 
```

Comment:
We don't have any other chapter split up in this manner, so lets stick with the standard. 

Leave 14.1 to 14.3 as they are.

Create a new chapter 15 header (15.0) and call it something like "Using Other Privacy Tools". It should generally match the style of 14.0, 13.0, and the other master chapters. It may be a lot briefer because we just have this one tool at this point. Maybe more in the future?

then, lay in your new section as 15.1.

Yeah, this is going to create a duplicate chapter 15. We'll get everything else renumbered at some point in the future.

Reply:
> Create a new chapter 15 header (15.0) and call it something like "Using Other Privacy Tools". It should generally match the style of 14.0, 13.0, and the other master chapters.

"Using i2p" will be better same as "Using Tor" because we are explaining similar thing here (creating hidden service for Bitcoin Core). I will have to add 1 to everything below [this](https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line#part-five-programming-with-rpc)

>  It may be a lot briefer because we just have this one tool at this point. Maybe more in the future?

Yes

> then, lay in your new section as 15.1

Okay. Will do this today.

> Yeah, this is going to create a duplicate chapter 15. We'll get everything else renumbered at some point in the future.

Or we can renumber in the same PR. Will be easy and its not code that will change any functionality for users, just the numbers in doc.

Reply:
Sounds great.

Reply:
Made changes in https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/commit/e6faf1cc07f4477c6508bbf82b1dc59940f101d9